sign-in.txt

```diff
@@ -5,3 +5,4 @@
         
        NAME        :   GitHub USERNAME 
 - Brian Bansenauer : Bansenauer-Cascadia
+- Gori Jesionowski : gjesionowski
```

Comment:
looks like you got the formatting correct here

Reply:
I have made the requested changes

Reply:
I thought resolving this conversation was the right move, but I think that was a mistake. Unresolved it and adding comment